index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Please I kindly suggest you to be more specific with the langueage that you´re using, in this case it would be something like this: 
`<html lang="en-US">`